Gemfile.jruby-3.1.lock.release

```diff
@@ -322,14 +314,13 @@ GEM
       lru_redux (~> 1.1.0)
     logstash-filter-drop (3.0.5)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-filter-elastic_integration (8.17.0-java)
+    logstash-filter-elastic_integration (8.17.1-java)
```

Comment:
We will discuss with @yaauie if we can release 9.0.0 plugin version and include in next BCs. For now this is not harmful.

Reply:
I believe we should release `9.0.0.alpha${X}` of that gem the day before each rc or bc, so that it includes the closest possible approximation of the Elasticsearch components as will ship in the rc or bc.